CHANGELOG.md

```diff
@@ -366,3 +366,117 @@ A wealth of bugfixes brought to you by rai, iota, and friends. You'll be rich. T
 - fixed deobfuscation level icons not immediately updating after stat generation completes (thanks [pitheguy](https://github.com/PiTheGuy)!)
 - improved wording of some save dialogs (thanks [pitheguy](https://github.com/PiTheGuy)!)
 - improved `PackageIndex` javadoc
+
+# 2.7.0
+
+Featuring code from supersaiyansubtlety, pitheguy, and rai, your recently bereaved QM team has fought through their grief over Quilt Mappings to bring you this release.
+Blood, sweat, refactors, enhancements, tears, unit tests, bugfixes, blood, and more have gone into this release.
+
+- added tooltips showing javadoc and declarations for entries ([#307](https://github.com/QuiltMC/enigma/pull/307))
+    - contains 3 elements:
+        - navigable links to parent classes
+            - clicking a class will open its tooltip, control-clicking a package will navigate to and expand it in the tree
+        - formatted javadoc
+        - snippet of code from the declaration
+    - tooltips are movable via dragging
+    - tooltips can be disabled or made non-interactable via the config
+- reworked commands and added a new search command
+    - added a new `search-mappings` command ([#282](https://github.com/QuiltMC/enigma/pull/282))
+        - allows searching for mappings using regular expressions. and normal expressions. as you desire
+        - includes extensive options for filtering by types, access, names, and more
+        - allows sorting by name, package, and depth
+    - added specification by name for command arguments (`parameter=value`) instead of positionally ([#280](https://github.com/QuiltMC/enigma/pull/280))
```

Comment:
```suggestion
    - added specification by name for command arguments (`parameter=value`) instead of by position ([#280](https://github.com/QuiltMC/enigma/pull/280))
```
sorry I know I suggested positionally but it sounded a little off in the re-word